Content.Server/Objectives/Systems/KillPersonConditionSystem.cs

```diff
@@ -86,34 +90,49 @@ private void AssignRandomTarget(EntityUid uid, ObjectiveAssignedEvent args, Pred
 
         _target.SetTarget(uid, _random.Pick(selectedHumans), target);
     }
-
+    // DeltaV - start making people only die once from EE
     private float GetProgress(EntityUid target, bool requireDead)
     {
         // deleted or gibbed or something, counts as dead
         if (!TryComp<MindComponent>(target, out var mind) || mind.OwnedEntity == null)
+        {
+            if (!requireDead && !_wasKilled.Contains(target)) _wasKilled.Add(target);
             return 1f;
+        }
 
         // dead is success
         if (_mind.IsCharacterDeadIc(mind))
+        {
+            if (!requireDead && !_wasKilled.Contains(target)) _wasKilled.Add(target);
```

Comment:
I think you should probably do a line break for these! Feels weird doing it on line line when its done with a line break everywhere else.

You also might want to remove the other check and only keep this one because of how the objective works now.